store.js

```diff
@@ -1,109 +1,100 @@
+    }
+    return userId;
+  };
+
+  const setAsync = promisify(client.set).bind(client);
   const getAsync = promisify(client.get).bind(client);
   const delAsync = promisify(client.del).bind(client);
-  const smembersAsync = promisify(client.smembers).bind(client);
-  const sremAsync = promisify(client.srem).bind(client);
 
-  const getUKey = userId => `${baseString}_${userId}`;
+  const getKeyPrefix = userId => `${userId}_${baseString}_`;
 
-  const getRTKey = refreshToken => `${baseString}_${refreshToken}`;
+  const getKey = (userId, refreshToken) => `${getKeyPrefix(userId)}${refreshToken}`;
 
   /**
    * Register token and refresh token to the user
    * @param {String} userId
    * @param {String} refreshToken
    * @param {String} accessToken
    * @param {Number} ttl time to live in ms
-   * @returns {Promise<Boolean>} returns true on success
+   * @returns {Promise<Boolean>} returns true when created.
    */
-  const registerTokens = (userId, refreshToken, accessToken, ttl) => {
-    RefreshToken(refreshToken);
-    const stringifiedValue = JSON.stringify({uid: UserId(userId), at: Token(accessToken)});
-    return new Promise((resolve, reject) => client
-    .multi()
-    // Sets refreshToken -> accessToken & userId
-    .set(getRTKey(refreshToken), stringifiedValue, 'EX', TTL(ttl))
-    // Adds refreshToken to userId set
-    .sadd(getUKey(userId), refreshToken)
-    .expire(getUKey(userId), TTL(ttl))
-    .exec(cbHandler(resolve, reject)))
-    .then(Boolean);
-  };
+  const registerTokens = (userId, refreshToken, accessToken, ttl) => setAsync(
+    getKey(
+      UserId(userId),
+      RefreshToken(refreshToken)
+    ),
+    Token(accessToken),
+    'EX', TTL(ttl)
+  ).then(Boolean);
 
   /**
-   * Returns the access token associated with the refresh token
+   * Returns the user's token using the userId and the refresh token
+   * @param userId
    * @param refreshToken
-   * @returns {Promise<String|undefined>} the token or undefined if not found.
+   * @returns {Promise<String>} the access token if found or null
    */
-  const getAccessToken = async refreshToken => {
-    const value = await getAsync(getRTKey(RefreshToken(refreshToken)));
-    if(value) {
-      return JSON.parse(value).at;
-    }
-    return undefined;
+  const getAccessToken = (userId, refreshToken) =>
+    getAsync(getKey(UserId(userId), RefreshToken(refreshToken)));
+
+  /**
+   * Scans for redis keys
+   * @param userId
+   */
+  const scanKeys = userId => {
+    const set = new Set();
+    return new Promise((resolve, reject) => {
+      redisScan({
+        redis: client,
+        pattern: `${getKeyPrefix(userId)}*`,
+        keys_only: false,
+        each_callback: function (type, key, subkey, length, value, cb) {
+          set.add(key);
+          cb();
+        },
+        done_callback: function (err) {
+          if(err) {return reject(err);}
+          return resolve([...set]);
+        }
+      });
+    });
```

Comment:
I don't remember this boilerplate being necessary. Maybe we can create an issue to switch to `ioredis`.